internal/src/components/modal-score-details.jsx

```diff
@@ -1,8 +1,55 @@
-import React from 'react'
+import './modal-score-details.scss'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
+import DataGridStudentScores from './data-grid-student-scores'
+import QuestionPreview from './question-preview'
+import InstructionsFlag from './instructions-flag'
 
-export default function ModalScoreDetails() {
-	return <div>@TODO</div>
+export default function ModalScoreDetails(props) {
+	const [previewType, setPreviewType] = useState(null)
```

Comment:
Instead I'm thinking of `const [selectedQuestion, setSelectedQuestion] = useState(null)`. Then if selectedQuestion is null show the instructions, otherwise, pass the selectedQuestion into <QuestionPreview>.

Reply:
Ah, well, I could see that being an issue with the attempt header row. What you have is good then, but still need to pass the selected question into <QuestionPreview>.